### PR TITLE
Add Claims Start Page

### PR DESCRIPTION
**Pull request by CatalinVoineag**
## Context

The start page will be the new root_path of the claims domain, some changes have been made to accomodate this like specs or redirects

## Changes proposed in this pull request

New root_path
New start page

## Guidance to review

Go to claims root_path
Inspect the start page
Click Start now
Sign in as before

## Screenshots

Prototype: https://itt-mentoring-beta-94341ca35a2c.herokuapp.com/

https://github.com/DFE-Digital/itt-mentor-services/assets/11318084/acfc1b27-09f1-4819-b54e-dfcf0bd2145d

### CODE REVIEW

**Comment by CatalinVoineag** [comment]
[Add start page](https://trello.com/c/ls6SvSrr/207-add-start-page)

File: app/views/claims/pages/start.html.erb [hunk 1]
```diff
@@ -0,0 +1,48 @@
+<div class="govuk-width-container">
```
Comment (JamieCleare2525) [inline app/views/claims/pages/start.html.erb:1]: Missing `<%= content_for :page_title, ... %>`

Comment (CatalinVoineag) [reply app/views/claims/pages/start.html.erb:1]: 🤦🏻

File: spec/system/claims/sign_in_page_spec.rb [hunk 2]
```diff
@@ -1,13 +1,13 @@
 require "rails_helper"
 
-RSpec.describe "Home Page", type: :system, service: :claims do
-  scenario "User visits the claims homepage with dfe sign in" do
+RSpec.describe "Sign in Page", type: :system, service: :claims do
+  scenario "User visits the claims sing in page with dfe sign in" do
```
Comment (JamieCleare2525) [inline spec/system/claims/sign_in_page_spec.rb:4]:
```suggestion
  scenario "User visits the claims sign in page with dfe sign in" do
```

File: spec/system/claims/support/schools/view_a_school_spec.rb [hunk 3]
```diff
@@ -5,8 +5,7 @@
   let!(:school) { create(:school, :claims) }
 
   scenario "View a school's details as a support user" do
-    user_exists_in_dfe_sign_in(user: support_user)
-    when_i_sign_in
+    sign_in_as(support_user)
```
Comment (JamieCleare2525) [inline spec/system/claims/support/schools/view_a_school_spec.rb:8]:
```suggestion
    when_i_sign_in_as(support_user)
    - or - 
    given_i_sign_in_as(support_user)
```

File: app/views/claims/pages/start.html.erb (hunk 1 above)
Comment (CatalinVoineag) [reply app/views/claims/pages/start.html.erb:1]: Oh, I didn't put it in as the default one is correct `Claim funding for mentors - GOV.UK`
Screenshot from 2024-03-08 14-52-11

**Review by JamieCleare2525 — APPROVED** [review]

**Pull request merged**